### PR TITLE
Add doc for sign-offs

### PR DESCRIPTION
**Pull request by flodolo**
I think this should be good for review.

Feel free to add more point in the review sections, if you can think of any.

### CODE REVIEW

File: products/review/signoffs.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+To perform the sign-off, click on the sparkline icon. It will open a new page, showing information about the last sign-off, and the changesets that follows it.
+
+![List of changesets](/assets/images/signoffs/changesets_table.png)
+
+For each changeset, it’s possible to check `compare-locale`’s results by clicking on the link in the second column from the left. Errors and warnings would be reported in the link too.
+
+Some changesets won’t have a link to `compare-locales`’s results (e.g. the second row in the image above). That indicates a changeset that doesn’t affect the product you’re currently signing off (e.g. a Thunderbird change in Firefox), and it’s not a good candidate for sign-offs.
+
+By clicking the top gray circle on the rightmost column, you can display a diff between the latest changeset and the most recent sign-off. Note that both circles can be dragged, to display the diff between specific changesets. It’s also possible to load more changeset at the bottom of the table using the *Load more* button.
```
Comment (Pike) [inline products/review/signoffs.md]: Actually, all of the column can be clicked to open the diff view. Yeah, magic.

**Review by Pike — CHANGES_REQUESTED** [review]
Can we fork this into two PRs?

I'd like to iterate over the string review part a bit more, but the sign-offs looks great already. There's just one nit about the magic of the magic of the diff UI, not sure if fixing that will improve the doc ;-)

**Comment by flodolo** [comment]
> Can we fork this into two PRs?

Sure. Let me keep this one for sign-offs, and split review in a different PR.

**Comment by flodolo** [comment]
Moved review.md in #118

**Review by Delphine — APPROVED** [review]
This looks good thanks. If ever I think of more things to add I edit this doc.

**Review by Pike — APPROVED** [review]
I think this is good.

There's a bit that I'm wondering about: the actual review process is just the last few bits on this page. So people trying to understand what we're doing here and why probably have their head off by the time they get there.

Maybe it makes sense to reorder this doc so that the actual review comes pretty early, and then the path towards the review page is explained?

If so, good for a follow-up.

**Comment by flodolo** [comment]
> Maybe it makes sense to reorder this doc so that the actual review comes pretty early, and then the path towards the review page is explained?

This is mostly for internal use (and curious folks). I'm afraid that putting that information without explaining first how diff view works, how to reach it, etc. might become really hard. Not sure if a reminder at the beginning would be enough.

I'm going to merge this, happy to iterate over the doc in the future. We'll likely come up with more checks too.

**Pull request merged**